fleetctl/ssh.go

```diff
@@ -131,6 +136,15 @@ func runSSH(args []string) (exit int) {
 	return
 }
 
+func findSSHPort(addr string) string {
+	SshPort := sharedFlags.SshPort
```

Comment:
It's unnecessary to copy `sharedFlags.SshPort` locally before using it below. Just get rid of this line.